code/implementation/docker/machine/src/main/java/io/cattle/platform/docker/machine/launch/AuthServiceLauncher.java

```diff
@@ -154,6 +155,7 @@ public String getPrivateKey() {
         list.add(SECURITY_SETTING);
         list.add(AUTH_SERVICE_LOG_LEVEL);
         list.add(AUTH_SERVICE_CONFIG_UPDATE_TIMESTAMP);
+        list.add(AUTH_SERVICE_REDIRECT_WHITELIST_SETTING);
```

Comment:
As mentioned on auth-service PR, we need to watch this setting and reload the auth-servers on change since auth-service is reading this from DB everytime anyways.